scanner/src/test/kotlin/ScanCodeTest.kt

```diff
@@ -101,7 +106,116 @@ class ScanCodeTest : WordSpec({
             val resultFile = File(javaClass.getResource(resultFileName).toURI())
             val result = ScanCode.getResult(resultFile)
             val summary = ScanCode.generateSummary(Instant.now(), Instant.now(), result)
+
             ScanCode.mapUnknownErrors(summary.errors) shouldBe false
         }
     }
+
+    "getRootLicense()" should {
+        "succeed for a result containing a LICENSE file" {
+            val resultFileName = "/oss-review-toolkit-license-and-readme_scancode-2.9.2.json"
+            val resultFile = File(javaClass.getResource(resultFileName).toURI())
+            val result = jsonMapper.readTree(resultFile)
```

Comment:
The lines above could go to a helper function because they are repeated in all tests.

Reply:
Actually, as the third line some tests have

    val result = jsonMapper.readTree(resultFile)

while others have

    val result = ScanCode.getResult(resultFile)

so we'd only safe two common lines. Not sure if that's worth a helper function.